### PR TITLE
fixes #17469 - add apipie-bindings examples to the API docs

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: _includes/manuals/1.13/5.1.9_apipie-bindings.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
+The following examples show how to do basic API operations using [apipie-bindings](https://github.com/Apipie/apipie-bindings).
+
+#### Show a Collection of Objects
+
+Get of a collection of domains: `GET /api/domains`
+
+Call the **index** function of the **domains** resource.
+
+```$ vim domains.rb```
```
Comment (mmoll) [inline _includes/manuals/1.13/5.1.9_apipie-bindings.md]: IMHO uneeded

**Review by mmoll — CHANGES_REQUESTED** [review]

File: _includes/manuals/1.13/5.1.9_apipie-bindings.md (hunk 1 above)
Comment (evgeni) [reply _includes/manuals/1.13/5.1.9_apipie-bindings.md]: the vim line? ack

**Comment by evgeni** [comment]
given that the 1.14 manual has been started, should I move this there?

**Comment by mmoll** [comment]
Copy it there, rather, we want this in both versions. :)

**Comment by evgeni** [comment]
ack, done :)

**Comment by mmoll** [comment]
merged, спасибо Женя!

**Pull request merged**